src/ScnSocialAuth/Options/ModuleOptions.php

```diff
@@ -1299,4 +1309,33 @@ public function getEnableSocialRegistration()
     {
         return $this->enableSocialRegistration;
     }
+
+    /**
+     * @return boolean
+     */
+    function getDebugMode() {
```

Comment:
Travis build is failing because your new methods have cuddled the brace. Move them all to a new line and it should go green.